src/cascadia/TerminalSettingsModel/ColorScheme.cpp

```diff
@@ -46,15 +47,22 @@ ColorScheme::ColorScheme() :
     _SelectionBackground{ DEFAULT_FOREGROUND },
     _CursorColor{ DEFAULT_CURSOR_COLOR }
 {
+    Utils::InitializeCampbellColorTable(_table);
+}
+
+ColorScheme::ColorScheme(winrt::hstring name) :
+    ColorScheme{}
+{
+    _Name = name;
 }
 
 ColorScheme::ColorScheme(winrt::hstring name, Color defaultFg, Color defaultBg, Color cursorColor) :
-    _Name{ name },
-    _Foreground{ defaultFg },
-    _Background{ defaultBg },
-    _SelectionBackground{ DEFAULT_FOREGROUND },
-    _CursorColor{ cursorColor }
+    ColorScheme{ name }

```

Comment:
Wait couldn't this still be an initializer list?
```suggestion
    ColorScheme{ name },
	_Foreground{ defaultFg },
    _Background{ defaultBg },
    _SelectionBackground{ DEFAULT_FOREGROUND },
    _CursorColor{ cursorColor }
```

my c++fu is not strong, but I thought this worked

Reply:
> a delegating constructor cannot have other mem-initializers

😢

Reply:
The correct way for you to do this is to delegate everything to the _largest_ constructor.